tune_sklearn/tune_search.py

```diff
@@ -221,8 +221,9 @@ class TuneSearchCV(TuneBaseSearchCV):
         max_iters (int): Indicates the maximum number of epochs to run for each
             hyperparameter configuration sampled (specified by ``n_trials``).
             This parameter is used for early stopping. Defaults to 1.
-            When using warm start to early stop on ensembles, this will
-            determine `n_estimators` for the final refitted ensemble.`
+            When using warm start to early stop on ensembles, each iteration
+            will fit `ensemble_estimator.n_estimators // max_iters` new
+            estimators.
```

Comment:
```suggestion
           Depending on the classifier type provided, a resource parameter
           (`resource_param = max_iter or n_estimators`) will be detected.
           The value of `resource_param` will be treated as a
           "max resource value", and all classifiers will be
           initialized with `max resource value // max_iters`, where 
           `max_iters` is this defined parameter. On each epoch, 
           resource_param (max_iter or n_estimators) is
           incremented by `max resource value // max_iters`.
```

Reply:
^ feel free to accept/or reject. 

I think this is not properly linted, so if accepting this change, be sure to run it through the linter!